tests/unit/panther_analysis_tool/test_main.py

```diff
@@ -94,14 +95,16 @@ def test_rules_from_current_dir(self):
         # This is a work around to test running tool against current directory
         return_code = -1
         invalid_specs = None
-        valid_rule_path = self.fixture_path + 'valid_analysis/policies'
+        valid_rule_path = os.path.join(FIXTURES_PATH, 'valid_analysis/policies')
         # test default path, '.'
         with Pause(self.fs):
             original_path = os.getcwd()
-            os.chdir(valid_rule_path)
-            args = pat.setup_parser().parse_args('test'.split())
-            return_code, invalid_specs = pat.test_analysis(args)
-            os.chdir(original_path)
+            try:
```

Comment:
This is a safeguard, in case an exception occurs in the following lines, the directory would not be switched back, which is a potential state leak for other tests.